### PR TITLE
Replace shell script with a python3 script

### PR DESCRIPTION
**Pull request by jmichelp**
Updated the shell script that generates the certificates and the
documentation accordingly.
Caveat: now installation is a 2-step procedure, installing OS and
application are split into 2 commands.

- [x] Tests pass
- [x] Appropriate changes to README are included in PR

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -261,5 +318,7 @@ sudo udevadm control --reload
 
 Then, you will need to unplug and replug the key for the rule to trigger.
 
+## Testing the key
+
```
Comment (gendx) [inline docs/install.md]: These changes overlap with https://github.com/google/OpenSK/pull/39. Let's keep them there.

File: README.md [hunk 2]
```diff
@@ -49,23 +49,22 @@ For a more detailed guide, please refer to our
...
-    installed, you can install both TockOS and the OpenSK application by running
-    the following command, depending on the board you have:
+2.  If Tock OS is already installed on your board, move to the next step.
+    Otherwise, just run one of the following commands, depending on the board
+    you have:
 
     ```shell
     # Nordic nRF52840-DK board
-    board=nrf52840dk ./deploy.sh os app
+    ./deploy.py os --board=nrf52840dk
```
Comment (ia0) [inline README.md]: Typo: nrf52840_dk

File: docs/install.md (hunk 1 above)
Comment (jmichelp) [reply docs/install.md]: Done

File: docs/install.md [hunk 3]
```diff
@@ -138,45 +141,72 @@ Our build script is responsible for converting `opensk_cert.pem` and
...
-       Compiling ctap2 v0.1.0 (.)
-         Finished release [optimized] target(s) in 7.60s
-     [STATUS ] Flashing binar(y|ies) to board...
-     [INFO   ] Using known arch and jtag-device for known board nrf52dk
-     [INFO   ] Finished in 0.305 seconds
-
-     [STATUS ] Installing app on the board...
-     [INFO   ] Using known arch and jtag-device for known board nrf52dk
-     [INFO   ] Finished in 0.975 seconds
+    $ ./deploy.py os --board=nrf52840_dongle
```
Comment (ia0) [inline docs/install.md]: Typo: nrf52840_dk

**Pull request merged**